nikola/plugins/task/indexes.py

```diff
@@ -95,7 +95,8 @@ def get_path(self, classification, lang, dest_type='page'):
                 self.site.config['RSS_PATH'](lang),
                 self.site.config['RSS_FILENAME_BASE'](lang)
             ], 'auto'
-        # 'page' (index) or 'feed' (Atom)
+        if dest_type == 'feed':
+            return [self.site.config['ATOM_PATH'](lang)], 'always'
```

Comment:
@felixfontein: why is page number support not implemented for feeds?

Reply:
Well, that would be a simple change. But:
 1. nobody asked for it so far (I think);
 2. there's still the big open question why the many .atom feed files are needed after all (#3016).
I'd like to see an answer to #3016 first.